src/app/shared/components/List/ListItem.tsx

```diff
@@ -199,12 +205,14 @@ export default function ListItem<T extends { id: string }>({
 
             function handleOpenPosition() {
               const element = menuButtonRef.current;
+              const contextMenu = menuItemsRef?.current?.offsetHeight || 300;
```

Comment:
this const has the `contextMenuHeight`, isn't it?

Reply:
Yep, I can update the name to a better one.